### ISSUE TITLE
"releasename" in latex_elements is not working

### ISSUE THREAD

**Issue by ianychoi** [sphinx-doc/sphinx#3392]
### Problem

According to http://www.sphinx-doc.org/en/1.5.2/config.html (even http://www.sphinx-doc.org/en/latest/config.html ),  #1782, and #1966, Sphinx supports to redefine releasename by specifying the name in latex_elements.

However, it does not work and using renewcommand for "releasename" also does not work.

From my investigation, nothing for making use of "releasename" is in sphinx/writers/latex.py.
Moreover, the result of renewcommand exists, however, the result is overrided by sphinx/templates/latex/content.tex_t 

#### Procedure to reproduce the problem

The following is the part from https://gist.github.com/ianychoi/91b97730f90bd1ffa61eff00c6ca273e#file-conf-py-L212
(modified from: http://git.openstack.org/cgit/openstack/openstack-manuals/tree/doc/arch-design/source/conf.py )
```
latex_elements = {
    'releasename': 'RELEASE_NAME_IN_LATEX_ELEMENTS',
    # The paper size ('letterpaper' or 'a4paper').
    'papersize': 'a4paper',

    # The font size ('10pt', '11pt' or '12pt').
    'pointsize': '11pt',

    'figure_align': 'H',

    # Additional stuff for the LaTeX preamble.
    'preamble': r"""
\makeatletter

\renewcommand{\releasename}{RELEASE_NAME_BY_RENEWCOMMAND}
""",
}
```
#### Error logs / results
- I cannot find "RELEASE_NAME_IN_LATEX_ELEMENTS" on the result .tex file.
Part of .tex file is:
```
\renewcommand{\releasename}{RELEASE_NAME_BY_RENEWCOMMAND}



\title{Architecture Design Guide}
\date{Feb 05, 2017}
\release{0.9}
\author{OpenStack contributors}
\newcommand{\sphinxlogo}{}
\renewcommand{\releasename}{Release}
\makeindex

```
The full tex result is available at
- https://gist.github.com/ianychoi/d0abaefd25912aef356786e5b71126f9

#### Expected results
I expect that "Release" string needs to be overrided finally in the tex file and after then successfully rendered into the PDF

### Reproducible project / your project
- I am now working https://review.openstack.org/#/c/427826 (related line: https://review.openstack.org/#/c/427826/7..8/doc/arch-design/source/conf.py ).

### Environment info (my local test environment)
- OS: Linux (Ubuntu 16.04 LTS)
- Python version: 2.7.12
- Sphinx version: 1.5.2
- <Extra tools e.g.: Browser, tex or something else>

**Comment by jfbu** [comment]
Thanks for report, in my short investigation the problem is in the latex writer which overrides the user setting and uses the translation of `Release` according to language.

Here is a temporary (in the sense of "will not be needed once the issue is fixed...", not in the sense "will stop working after a while") work-around:
```
    'preamble': r"""
\makeatletter
\renewcommand{\release}[1]{%
  \renewcommand{\py@release}{RELEASE\_NAME\_BY\_RENEW\_COMMAND\space\version}%
  \renewcommand{\version}{#1}}
\makeatother
"""
```
You must TeX-escape the chosen release name. Can you try it ?

**Comment by ianychoi** [comment]
@jfbu I confirm that your workaround works well. Thanks a lot for this workaround and also your fix on upstream!

**Comment by jfbu** [comment]
Thanks for report and feedback ! Next 1.5.3 release fixes it: but I will add something to the docs to recall the TeX must be escaped, like in your example the underscores.

Closing now, thanks !

Fixed at 20dc524 and docs updated at 80e39e4
